extensions/interactions/MathEquationInput/directives/oppia-interactive-math-equation-input.component.ts

```diff
@@ -62,7 +62,7 @@ export class InteractiveMathEquationInput implements OnInit {
     private translateService: TranslateService,
   ) {}
 
-  isCurrentAnswerValid(checkForTouched = true): boolean {
+  isCurrentAnswerValid = (checkForTouched = true): boolean => {
```

Comment:
Ditto: function definition syntax.

Ditto elsewhere.

Reply:
Same reason as above. `isCurrentAnswerValid` is passed as a callback to `this.currentInteractionService.registerCurrentInteraction`.

Reply:
done